src/docker/DockerCompose.js

```diff
@@ -200,6 +214,21 @@ class DockerCompose {
       .filter(Boolean);
   }
 
+  /**
+   * Invert list of Docker service names
+   *
+   * @private
+   * @param {Object} envs
+   * @param {string} [invertServiceName]
+   * @return {string[]}
+   */
+  async invertContainerNamesList(envs, invertServiceId = undefined) {
```

Comment:
Pretty strange function for this class. Do you use it many times in different places?

Reply:
Moved logic to status command